github/enterprise_actions_runners.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CreateRegistrationToken creates a token that can be used to add a self-hosted runner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/actions/#create-a-registration-token-for-a-repository
+func (s *EnterpriseService) CreateRegistrationToken(ctx context.Context, enterprise string) (*RegistrationToken, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners/registration-token", enterprise)
+
+	req, err := s.client.NewRequest("POST", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	registrationToken := new(RegistrationToken)
+	resp, err := s.client.Do(ctx, req, registrationToken)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return registrationToken, resp, nil
+}
+
+// ListRunners lists all the self-hosted runners for a enterprise.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/enterprise-admin/#list-self-hosted-runner-groups-for-an-enterprise
+func (s *EnterpriseService) ListRunners(ctx context.Context, enterprise string, opts *ListOptions) (*Runners, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/actions/runners", enterprise)
```

Comment:
There seems to be a mismatch here too.  The documentation link on line 35 points to an endpoint that looks like this:

```
GET /enterprises/{enterprise}/actions/runner-groups
```

and not the one listed on line 37.